### PR TITLE
Add vim and nano to WLoopOS

### PR DESCRIPTION
**Pull request by ethanckim**

### CODE REVIEW

File: wloop_os/configs/rpi4_defconfig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 BR2_arm=y
 BR2_cortex_a72=y
 BR2_ARM_FPU_NEON_VFPV4=y
-BR2_JLEVEL=8
```
Comment (ethanckim) [inline wloop_os/configs/rpi4_defconfig:4]: I think that it is more optimized for buildroot to automatically decide the # of jobs to run simultaneously, not restricted to 8.

File: wloop_os/configs/rpi4_defconfig [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+BR2_CCACHE=y
```
Comment (ethanckim) [inline wloop_os/configs/rpi4_defconfig:4]: Enables compiler cache for faster build times

File: wloop_os/configs/rpi4_defconfig [hunk 3]
```diff
@@ -19,6 +19,7 @@ BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="$(BR2_EXTERNAL_WLoopOS_PATH)/board/rpi4/ker
 BR2_LINUX_KERNEL_DTS_SUPPORT=y
 BR2_LINUX_KERNEL_INTREE_DTS_NAME="bcm2711-rpi-4-b"
 BR2_LINUX_KERNEL_NEEDS_HOST_OPENSSL=y
+BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
```
Comment (ethanckim) [inline wloop_os/configs/rpi4_defconfig:22]: Shows packages that are also provided by busybox. Needed to enable vim in the menuconfig GUI.

**Comment by ethanckim** [comment]
Please review the comments I have appended to this PR

**Review by RyanJMah — APPROVED** [review]
Lgtm, nice work!

**Comment by ethanckim** [comment]
Closed PR as develop branch has been created

**Pull request closed without merging**